ui/src/containers/Owner/CreatedAt.js

```diff
@@ -1,6 +1,13 @@
 import moment from 'moment';
 
-export default (createdAt) => {
+export default (createdAt, {
+  future = 'Created',
```

Comment:
If these both default to `Created` I don't really understand why this is needed.

Reply:
Just seen why this has been done, in another comment I've said to revert this and make a new function called `expiresAt` so that it makes more sense semantically when you read the code.